docker/oso-rhel7-zagg-client/root/config.yml

```diff
@@ -25,6 +26,9 @@
     - name: run pcp checks every 5 minutes
       minute: "*/5"
       job: /usr/bin/ops-zagg-client --send-pcp
+    - name: run pcp sampler every 5 minutes
+      minute: "*/5"
+      job: /usr/local/bin/pcp_sample.py -m kernel.all.cpu.idle,kernel.all.cpu.nice,kernel.all.cpu.steal,kernel.all.cpu.sys,kernel.all.cpu.user,kernel.all.cpu.wait.total
```

Comment:
Once pcp_sample.py is packaged, it shouldn't be in /usr/local/bin, so that'll need to be fixed.